clients/docs/includes/certs-truststore.rst

```diff
@@ -0,0 +1,68 @@
+Configure SSL trust store
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Depending on your operating system or Linux distro you may need to take extra
+steps to set up the SSL CA root certificates. If your system doesn't have the
+SSL CA root certificates properly set up, you may receive an error message
+similar to the following:
+
+.. code-block:: bash
+
+   %3|1554125834.196|FAIL|rdkafka#producer-2| [thrd:sasl_ssl://pkc-epgnk.us-central1.gcp.confluent.cloud\:9092/boot]: sasl_ssl://pkc-epgnk.us-central1.gcp.confluent.cloud\:9092/bootstrap: Failed
+   %3|1554125834.197|ERROR|rdkafka#producer-2| [thrd:sasl_ssl://pkc-epgnk.us-central1.gcp.confluent.cloud\:9092/boot]: sasl_ssl://pkc-epgnk.us-central1.gcp.confluent.cloud\:9092/bootstrap: Faile
+   %3|1554125834.197|ERROR|rdkafka#producer-2| [thrd:sasl_ssl://pkc-epgnk.us-central1.gcp.confluent.cloud\:9092/boot]: 1/1 brokers are down
+
+In this case, you need to manually install a bundle of validated CA root certificates before running the client code.
+(For more information, see the documentation for `librdkafka <https://github.com/edenhill/librdkafka/wiki/Using-SSL-with-librdkafka>`__ on which this client is built)
```

Comment:
That link is how to generate your own private CA certs, reference the manual instead:
https://github.com/edenhill/librdkafka/blob/master/INTRODUCTION.md#ssl

Reply:
Updated link